Tests/Application/config/config_website.yml

```diff
@@ -3,13 +3,13 @@ parameters:
     database.url: '%env(resolve:DATABASE_URL)%'
 
 framework:
-    router: { resource: "%kernel.root_dir%/config/routing_website.yml" }
+    router: { resource: "%kernel.project_dir%/config/routing_website.yml" }
 
 security:
     access_decision_manager:
         strategy: affirmative
 
-    encoders:
```

Comment:
@alexander-schranz can we drop 7.2 support? Otherwise we have to duplicate the config files for 7.2 😅

Reply:
as discussed, it is a symfony issue not 7.2, therefore we have to add a bridge anyway 😅

Reply:
@Prokyonn I think you maybe possible to remove this lines and the SuluTestBundle already loads then correctly 5.4 or 6.0 via: 

https://github.com/sulu/sulu/blob/9361c365670e4b540897354b1c104ced1b0d0368/src/Sulu/Bundle/TestBundle/Kernel/SuluTestKernel.php#L132

Reply:
this was only added in sulu >2.5, I've added a bridge with two config files for the security part